qemu/tests/hpt_miscellaneous.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+def set_hpt(session, test, hpt_size):
+    """Run this step to set HPT in guest"""
+    set_cmd = "echo %s > /sys/kernel/debug/powerpc/hpt_order"
+    try:
+        st, o = session.cmd_status_output(set_cmd % (hpt_size))
+        # Get output after set command,some errors are there but
+        # they are acceptable
+        n_output = str(o)
+    except Exception as detail:
```

Comment:
What kind of `Exception` will be catched here? It seems you don't need to catch it as `detail`.

Reply:
Thanks and updated it already,could you please have a look again ?